mqtt/mqttd/src/tracing/edgehub.rs

```diff
@@ -13,11 +14,16 @@ pub fn init() {
         .or_else(|_| EnvFilter::try_from_default_env())
         .unwrap_or_else(|_| EnvFilter::new("info"));
 
+    let filter = log::LevelFilter::from_str(log_level.to_string().as_str())
```

Comment:
it captures logs as tracing and if the level is all lowercase it works as expected. The issue is when level is Debug for example it will log everything as trace, but the issue is not because of this change, it seems to be in EnvFilter